extra-cpp-bindings/src/lib.rs

```diff
@@ -659,4 +740,15 @@ mod test {
         .expect("blockscout query");
         assert_eq!(actual, expected);
     }
+
+    #[test]
+    pub fn test_get_token_holders() {
+        let actual_result = get_token_holders(
+            "https://blockscout.com/xdai/mainnet/api",
+            "0xed1efc6efceaab9f6d609fec89c9e675bf1efb0a",
+            1,
+            10,
+        );
+        println!("{:?}", actual_result);
```

Comment:
assert the result?

Reply:
ok

Reply:
The result (address and the balance) is likely to be changed. So it is difficult to assert the content.

Reply:
I'll still add it.

Reply:
done